combat-trainer.lic

```diff
@@ -2555,14 +2556,28 @@ class TrainerProcess
       pray_mat(game_state)
     when /^Recall$/i
       bput("recall #{game_state.npcs.first}", 'Roundtime', 'You are far too occupied') unless game_state.npcs.empty?
-    when /^Barb Research Augmentation$/i
-      bput('meditate research monkey', 'You clear ')
-    when /^Barb Research Warding$/i
-      bput('meditate research turtle', 'You clear ')
-    when /^Barb Research Debilitation$/i
-      bput('meditate research earthquake', 'You clear ')
-    when /^Barb Research Utility$/i
-      bput('meditate research prediction', 'You clear ')
+    when /^Barb Research/i
+      # Note, the above regex is a 'starts with' match, don't append $ character to it
+      # so that we handle all scenarios of 'Barb Research' and 'Barb Research <magic skill>'.
+      #
+      # Debilitation cannot be learned via meditate research
```

Comment:
What do you think about adding a validator that Debilitation doesn't work now? I only suggest it because it is a change in behavior

Reply:
Done via https://github.com/rpherbig/dr-scripts/pull/5775/commits/ee8ed8a9d815a3c7a5f27a9b492db876ea57fbe5